src/main/java/com/jland/repository/impl/ConferenceRepositoryImpl.java

```diff
@@ -0,0 +1,28 @@
+package com.jland.repository.impl;
+
+import com.jland.model.Conference;
+import com.jland.repository.ConferenceRepository;
+import com.jland.repository.mapper.ConferenceMapper;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class ConferenceRepositoryImpl implements ConferenceRepository {
+
+    private static final ConferenceMapper conferenceMapper = new ConferenceMapper();
+    private static final String GET_CONFERENCE_SQL = "select id, description, date, address from conference where id = :id;";
```

Comment:
we can use query configuration class for all queries or xml if you config beans via xml